app/javascript/css/main.scss

```diff
@@ -515,3 +523,29 @@ a.disabled {
   color: var(--primary-color);
   border-bottom: 1px dashed var(--primary-color);
 }
+
+.nav-stacked li {
```

Comment:
is this the result of a faulty rebase? looks like it's a repeat definition from above in this file